server/views/formPages/incident/evidence.html

```diff
@@ -335,7 +293,7 @@ <h1 class="govuk-heading-xl mainHeading">Evidence</h1>
   <script src="/assets/js/jquery.min.js"></script>
   <script src="/assets/add-another.js"></script>
   <script>
```

Comment:
Do we need to do a document ready here? or is the order of the scripts enough?

Reply:
Not sure? we haven't had any issues with this so far